### PR TITLE
Feature/jcs/bug spray

### LINKED ISSUE #23: Connect Api to web-app
### Feature Categories:

- [x] Angular
- [ ] SCSS / Styling
- [ ] Testing
- [ ] UI 
- [ ] BugFix

**Describe the feature/Bug your gonna work on**

Connect the api to the web-app with angular pipes

**Additional context**
Add any other context or screenshots about the feature.

### LINKED ISSUE #49: Update openDialog() to take in dynamic data for the modal
### Feature Categories:

- [x] Angular
- [x] SCSS / Styling
- [ ] Testing
- [ ] UI 
- [ ] BugFix

**Describe the feature/Bug your gonna work on**

Modal is displaying but needs to be able to dynamically display information
Pass in data from api to test dynamic data being passed in.

**Additional context**
Add any other context or screenshots about the feature.

### LINKED ISSUE #50: Run Tests on Application to find bugs
### Feature Categories:

- [x] Angular
- [ ] SCSS / Styling
- [ ] Testing
- [ ] UI 
- [ ] BugFix
- [x] Application bug finding

**Describe the feature/Bug your gonna work on**

**Additional context**
Add any other context or screenshots about the feature.

### LINKED ISSUE #51: Fix any styling bugs found after app testing
### Feature Categories:

- [x] Angular
- [x] SCSS / Styling
- [ ] Testing
- [ ] UI 
- [ ] BugFix
- [x] Bug finder

**Describe the feature/Bug your gonna work on**

- Fix any styling issues that may happen from multiple sized screens
- Header styling break does get weird at different screen sizes 
- Document any fixes and get app ready for DEMO

**Additional context**
Add any other context or screenshots about the feature.

### LINKED ISSUE #60: Add DemoData for better user experience
### Feature Categories:

- [x] Angular
- [ ] SCSS / Styling
- [ ] Testing
- [ ] UI 
- [x] BugFix

**Describe the feature/Bug your gonna work on**
Final bug run through with modal component
Clean up any lingering styling issues

**Additional context**
Add any other context or screenshots about the feature.

### PR DESCRIPTION
**Pull request by JoshSevy**
# Turing Index Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Added in DemoData to give a better user experience when demoing the application
FIx critical error with modal component not recognizing dynamic data after reload
Connect api to make application fully functional

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Testing (add-testing to project)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:

There wasn't much data in the api for us to render to the user, I decided to add a demo data file for a better user experience with the application.

## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
ng test
ng lint
# Message for reviewer:
There wasn't much data to display from the api. I hard coded in some demo data would love you both to read through and help fine tune it for our presentation tomorrow.
# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs - Console log left in to show data coming back from api
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/shared/DemoData.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
+export const frontEndMod1 = {
```
Comment (Leasw144) [inline src/shared/DemoData.ts:1]: Great job doing the mock data

**Review by Leasw144 — APPROVED** [review]
Great job mocking the data for the demo

File: src/shared/DemoData.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+export const community = {
+  title: 'Guest Speaker: Gary Flowers ',
+  description: 'Over 25 years of experience in the technology industry, Gary has focused on enabling business process and innovation from a Senior and Executive level vantage point. Gary has worked in multiple industries both domestic and globally to drive increased profit and process efficiency. ',
+  zoomLink: 'https://turingschool.zoom.us/j/96910748883',
+  lessonPlan: 'https://www.linkedin.com/in/garyflowers/'
+};
+
+export const kitchen = {
+  title: 'Lets get Social',
+  description: 'Need a break, need someone to bounce your crazy ideas off. Come to the kithen and connect with fellow students, and staff. No Idea is too crazy here.',
```
Comment (NicoleLatifi) [inline src/shared/DemoData.ts]: Let's substitute "wild" instead of "crazy".

**Review by NicoleLatifi — APPROVED** [review]
Looks great! Thanks, Josh.

**Pull request merged**